### PR TITLE
chore: refactor to Deno project

### PR DESCRIPTION
**Pull request by KnorpelSenf**
This is sort of a Node project that works on Deno using `esm.sh`. That is not in line with the rest of the ecosystem, and it hinders us from building good tooling around this plugin.

This PQ refactors this project to be a Deno-first project.

### CODE REVIEW

**Comment by roziscoding** [comment]
WDYT about using dnt for this instead of deno2node in preparation for the migration to @grammyjs/build?

File: deno.jsonc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "lock": false,
+    "fmt": {
+        "proseWrap": "preserve",
+        "indentWidth": 4
```
Comment (roziscoding) [inline deno.jsonc:5]:
```suggestion
        "indentWidth": 4,
        "lineWidth": 120
```

The default value of 80 doesn't make a lot of sense with modern screens being easily able to fit more, and it leads to horrible looking lines that get broken in the middle with absolutely no need

**Review by roziscoding — APPROVED** [review]
Approving since my suggestions are non blocking. Feel free to ignore them at will.

**Comment by KnorpelSenf** [comment]
> WDYT about using dnt for this instead of deno2node in preparation for the migration to @grammyjs/build?

There are no plans to move to dnt.

File: deno.jsonc (hunk 1 above)
Comment (KnorpelSenf) [reply deno.jsonc:5]: I often have two source files open next to each other and 80 characters is a sane limit for me.

**Pull request merged**